app/api/dao/user.py

```diff
@@ -143,7 +143,7 @@ def get_user_by_username(username: str):
         return UserModel.find_by_username(username)
 
     @staticmethod
-    def list_users(user_id: int, search_query: str = "", is_verified = None):
+    def list_users(user_id: int, search_query: str = "", page: int = 0, size: int = 10, is_verified = None):
```

Comment:
Looks perfect. Though this is just a preference, i think you should change the name from `size` to `per_page` or something similar for readability. Someone might be tempted to ask, `size of what?`.

Reply:
Sounds good - changed to `per_page` :)